.github/workflows/test.yml

```diff
@@ -1,13 +1,13 @@
 name: Tests
 
-on: [push, pull_request]
+on: [pull_request]
 
 jobs:
   unit-test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        module: [tracking, messagingpush, messaginginapp, base]
+        module: [tracking, messagingpush, messaginginapp, base] # need to add datapipelines, here, this CI task is broken right now.
```

Comment:
From looking at the [latest failed CI job](https://github.com/customerio/customerio-android/actions/runs/8751552838/job/24017378777?pr=302), my guess at the CI failing is because the datapipelines module currently has on test functions in it. No tests run -> no code coverage report made -> no junit report made. So the actions are failing as they expect to see these reports. 

I would be OK with making a separate ticket if we have to. I would prefer if this PR had the CI fully setup and working, ready for us to do development. One idea to make that happen is create 1 placeholder test function in the CDP module. 1 test function that would pass. Then, the reports should generate as expected and pass the CI. We can delete this placeholder test when we begin the next ticket.

Reply:
If you look into the `test` action. 
<img width="288" alt="Screenshot 2024-04-19 at 10 01 14 PM" src="https://github.com/customerio/customerio-android/assets/4658866/1b1f7138-b2c3-40eb-ac9a-6118c666fb61">
You would see all of them are skipped, it only runs for `Base` which is last in list.